python_modules/dagster/MANIFEST.in

```diff
@@ -13,3 +13,7 @@ recursive-include dagster/_generate/templates *
 recursive-include dagster *.proto
 global-exclude __pycache__
 global-exclude *.py[co]
+global-exclude *.g4
+global-exclude *.interp
+global-exclude *.tokens
+global-exclude Makefile
```

Comment:
🤔 not sure about this Makefile exclude

Reply:
I mean I don't know why it's not already excluded, should double check with someone more familiar with it if that's an intentional decision or not.

Reply:
Checked with @benpankow, I think it's just because we didn't have any Makfiles in our python modules previously. Should be fine to exclude.